src/lib/geoip/providers/ip2location.ts

```diff
@@ -59,11 +60,11 @@ export const ip2LocationLookup = async (addr: string): Promise<Ip2LocationBundle
 		longitude: result.longitude ?? 0,
 		network: result.as ?? '',
 		normalizedNetwork: normalizeNetworkName(result.as ?? ''),
+		isHosting: result.usage_type ? HOSTING_USAGE_TYPES.includes(result.usage_type) : null,
 	};
 
 	return {
 		location,
-		isHosting: result.usage_type ? HOSTING_USAGE_TYPES.includes(result.usage_type) : undefined,
```

Comment:
No objections here, just curious about the reason for this change.

Reply:
`ProbeLocation` type already has `isHosting` field, so no need to store it separately.

Reply:
Ahah 😄. It is actually there in master: https://github.com/jsdelivr/globalping/blob/master/src/probe/types.ts#L13
I moved it from `ProbeLocation` to `LocationInfo`. Idea is to have one type for location that provider is returning, and one for location that is stored in probe. Ideally `isProxy` should be moved to `LocationInfo` too with default `null` value. So mutants like `Ip2LocationBundledResponse` can be removed.